docs/sources/administration/configuration.md

```diff
@@ -803,6 +803,57 @@ Use the [List Metrics API](https://docs.aws.amazon.com/AmazonCloudWatch/latest/A
 
 <hr />
 
+## [azure]
+
+Grafana supports additional integration with Azure services when hosted in Azure cloud.
+
+### cloud
+
+Azure could environment in which Grafana is hosted:
+
+| Azure Cloud                                      | Value                  |
+| ------------------------------------------------ | ---------------------- |
+| Microsoft Azure public cloud                     | AzureCloud (*default*) |
+| Microsoft Chinese national cloud                 | AzureChinaCloud        |
+| Microsoft German national cloud ("Black Forest") | AzureUSGovernment      |
+| US Government cloud                              | AzureGermanCloud       |
```

Comment:
Did these get switched? US Government cloud should be mapped to AzureUSGovernment and Microsoft German national cloud ("Black Forest") should be mapped to AzureGermanCloud.

Reply:
Yes, thanks for pointing out.